conan/tools/gnu/autotoolstoolchain.py

```diff
@@ -203,7 +203,7 @@ def _get_argument(argument_name, cppinfo_name):
                                        _get_argument("includedir", "includedirs"),
                                        _get_argument("oldincludedir", "includedirs"),
                                        _get_argument("datarootdir", "resdirs")])
-        return configure_install_flags
+        return [el for el in configure_install_flags if el]
```

Comment:
When empty "cpp.package.resdirs" this caused (harmful but weird) `''` configure argument.